### ISSUE TITLE
#old count

### ISSUE THREAD

**Issue by fgandellini** [cirpo/hubot-save-links#32]
It would be nice if the bot can say how many times a link has been posted, and how many times a user gets an #old warning.

**Comment by cirpo** [comment]
@fgandellini yes! I was thinking about about it last night! And at the end of the year we can even give a prize to whom had posted the highest number of old links :dancer: 

I was thinking about changing the storage, at the moment we are using redis but is becoming cumbersome to have all those relations: I'm thinking about switching to sqlite. I will open another issue about that so we can discuss it there.

**Comment by MicheleBertoli** [comment]
+1
